### ISSUE TITLE
Php 7.0^ compatibility

### ISSUE THREAD

**Issue by pschoffer** [Automattic/vaultpress#110]
Hi there,

running PhPCompatibility on vaultpress codebase reveals two warnings. Those don't seem to be very impactful issues, but it would be nice to get them addressed if possible.

```
$ phpcs --standard=PHPCompatibility --runtime-set testVersion 7.0- .

FILE: vaultpress.php
------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------
 3050 | WARNING | Function get_magic_quotes_gpc() is deprecated since PHP 7.4
------------------------------------------------------------------------------


FILE: cron-tasks.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 15 | WARNING | Since PHP 7.0, functions inspecting arguments, like func_get_args(), no longer report the original value as passed to a parameter, but will instead provide the current value. The parameter
    |         | "$tag" was used, and possibly changed (by reference), on line 14.
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by GaryJones** [comment]
We've now got WordPress VIP customers who are hesitant to jump to PHP 7.4, because of the deprecation notice that comes from the vip-go-mu-plugins repository from VaultPress.

(Also, some customers are also testing locally with PHP 8, so could also do with this being resolved.)

@jeherve Can this be addressed please, so that a new version of VaultPress can be pulled into vip-go-mu-plugins, and our customers can feel confident that we're more ready for PHP 8?

**Comment by jeherve** [comment]
cc @vianasw, who should be in a better position to help.

**Comment by vianasw** [comment]
Sorry, this ping fell through the cracks. I'll take a look at it.

**Comment by jeherve** [comment]
This was fixed in #123.
